lib/monitor/stats.go

```diff
@@ -231,6 +231,12 @@ func (m *Measurement) Update(measure Measure) {
 	m.System.Store(measure.CPUTimeSys)
 }
 
+func (m *Measurement) Aggregate() {
```

Comment:
Add a short comment - btw: I'm always lost when I have to search for functions that are named the same way for different structures. Is there a way to rename *Measurement.Aggregate* or *Stats.Aggregate*? And is *Aggregate* really the right word? Look at http://www.thesaurus.com/browse/aggregate for other words.